src/ngraph/runtime/executable.cpp

```diff
@@ -108,6 +108,11 @@ const ngraph::ResultVector& runtime::Executable::get_results() const
     return m_results;
 }
 
+size_t runtime::Executable::get_default_pipeline_depth() const
+{
+    return m_default_pipeline_depth;
```

Comment:
Just return the number here. It is a virtual function so a user can override it and change the value without the need for m_default_pipeline_depth